### ISSUE TITLE
[Enhancement] SwipeView change LeftItems/RightItems to Primary/Secondary & Support RTL

### ISSUE THREAD

**Issue by darrabam** [xamarin/Xamarin.Forms#10366]
## Summary
Current design of SwipeView mandates setting direction (LeftItems, RightItems ...etc.). However, this gets messy when dealing with applications that support both RTL & LTR languages. 

Setting page flow direction to RTL doesn't swap the SwipeView direction which make the location of the swap menu in the opposite direction of what is expected. Which makes the only option to support LTR & RTL is a lot of code behind. 

## API Changes

Change the terminology of LeftItems, RightItems to PrimaryItems /SecondaryItems (or better notion) and make the SwipeView sensitive to flow direction such that: 
- on LTR,  Primary menu appears by swiping left to right. 
- on RTL,  Primary menu appears by swiping right to left.

## Intended Use Case
This will significantly simplify using SwipeView in applications that supports both RTL & RTL languages and use SwipeView. By having this feature implemented, there will be no need to implement and add SwipeViews in the code behind.

**Comment by Tommigun1980** [comment]
I consider it a bug that the swipe menu doesn't work with RTL rather than an enhancement request, but that's just me.

**Comment by hartez** [comment]
Where do the top/bottom items fit it? Do we call them primary/secondary/top/bottom?

Also, how should we handle multiple items in a set? You can have more more than one item in LeftItems - do those also reverse direction? Or do we preserve the order in which they are added to the collection?

**Comment by darrabam** [comment]
@hartez yes, name them primary/secondary/top/bottom (or if you have a better name). The confusion today is with left/right and flow direction rtl/ltr.

 _>> how should we handle multiple items in a set_
Keep the behavior consistent with collectionview in horizontal mode when switching page/control to rtl. The I see it logical is as follows:
![image](https://user-images.githubusercontent.com/33980667/93986621-3fc13a00-fd8f-11ea-9e7e-af78519975b6.png)
